tsl/test/expected/dist_api_calls.out

```diff
@@ -216,44 +216,278 @@ _timescaledb_internal._dist_hyper_1_8_chunk
  
 (1 row)
 
--- Simple test of time_bucket_gapfill, which is disabled for now.
-\set ON_ERROR_STOP 0
+-- Simple tests of time_bucket_gapfill
 SELECT time_bucket_gapfill('3 hours', time, '2017-01-01 06:00', '2017-01-02 18:00'),
        first(value, time),
        avg(value)
 FROM disttable
 GROUP BY 1;
-ERROR:  time_bucket_gapfill not implemented for distributed hypertable
+     time_bucket_gapfill      | first |   avg   
```

Comment:
This test file (`dist_api_calls.c`) seems like the wrong place to test gapfill queries on distributed hypertables. I could maybe understand that the previous block functionality was here, but actual query output should probably be in `dist_query` or similar query-related test.

Reply:
@erimatnor I agree I want to create separate test, which will also compare query results between single node, non-partitionwise distributed and partitionwise distributed. I feel it is good to make as a separate commit and may be a separate PR.

Reply:
I guess that's fine, but is there a reason why it isn't possible to just move this to, e.g., `dist_query` right now and at least get some of those benefits?

Reply:
Imho it is double work. I investigated previously and haven't found a natural place to put this test. So it will require some work.